compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -528,19 +528,24 @@ impl<'ll, 'tcx> MiscMethods<'tcx> for CodegenCx<'ll, 'tcx> {
         if let Some(llpersonality) = self.eh_personality.get() {
             return llpersonality;
         }
+
+        let name = if wants_msvc_seh(self.sess()) {
+            Some("__CxxFrameHandler3")
+        } else if wants_wasm_eh(self.sess()) {
+            Some("__gxx_wasm_personality_v0")
```

Comment:
Will this need to be defined in the standard library or is it just a name that indicates wasm exceptions should be used and doesn't result in any runtime reference to it? If the former, would it be possible to give it a rust specific name? And if the latter can you add a comment to that extend?

Reply:
There is no need to define it in the standard library - LLVM tests specifically for the name of the personality function to determine the correct exception handling scheme (https://github.com/llvm/llvm-project/blob/4aa1cadf3c67c61b394051630896f06f35b562ca/llvm/lib/IR/EHPersonalities.cpp#L43)

This function will never be called and there is no need to define it anywhere, just the name is important.

I will add a comment 👍